tests/tests/src/core/Core_ErrorTests.mjs

```diff
@@ -25,7 +26,7 @@ function panicTest() {
       43
     ],
     "Should resolve test"
-  ], caught, (prim0, prim1) => prim0 === prim1, "Panic! uh oh");
```

Comment:
Why is this changed?

Reply:
Because I did not adapt the source like in the other examples. Changed now.